app/models/talk.rb

```diff
@@ -49,14 +49,15 @@ class Talk < ApplicationRecord
     attribute :speakers do
       speakers.pluck(:name)
     end
-    searchable_attributes [:title, :description]
+    add_attribute :_vector do
+      _vector # doesn't work yet
+    end
+    searchable_attributes [:title, :description, :_vector]
     sortable_attributes [:title]
 
     attributes_to_highlight ["*"]
   end
 
-  meilisearch enqueue: true
```

Comment:
@adrienpoly this was actually overriding the previous meilisearch block definition (I spent a while understanding why my `_vector` attribute wouldn't index 😅 )